cliconfig/config.go

```diff
@@ -178,7 +189,7 @@ func Load(configDir string) (*ConfigFile, error) {
 	}
 
 	// Can't find latest config file so check for the old one
-	confFile := filepath.Join(homedir.Get(), oldConfigfile)
+	confFile := getDefaultConfigDir(oldConfigfile)
```

Comment:
> Perhaps we should keep this one as it was;

mmm this one also resolves the config from the home dir, why don't we want to change it?